packages/app/src/systems/Overlay/components/OverlayDialog/OverlayDialog.tsx

```diff
@@ -19,7 +19,6 @@ export function OverlayDialog() {
       <Dialog.Content css={styles.content}>
         {/* Accounts */}
         {overlay.is('accounts.list') && <Accounts />}
-        {overlay.is('accounts.add') && <AddAccount />}
```

Comment:
should also remove leftovers from 'accounts.add'.
also overlay events to call `openAccountsAdd`